order/buyer/reissue.md

```diff
@@ -0,0 +1,49 @@
+---
+description: How to reissue an order as a buyer
+---
+
+# Reissue order
+
+As buyer you can send either a [new](issue.md) or updated purchase order to Tradecloud.
+
+## Send an updated order to Tradecloud
+
+As a buyer you can send an updated purchase order to Tradecloud.
+
+{% hint style="danger" %}
+Most supplier ERP integrations do not have the capability to automatically process an updated order. It will processed manually by the supplier and the order will have a longer human response time.
+{% endhint %}
+
+{% hint style="info" %}
+If an order line has order process status `Issued` or `In Progress` it will be `Reissued` and keep the same status.
```

Comment:
I think this might be confusing for some: will the order be `Reissued`, `Issued` or `In Progress`.
Proposed: "If an order line has order process status `Issued` or `In Progress` and it is reissued, it will keep the same status."

Same for order lines.

Reply:
Done 3x